crates/bevy_app/src/lib.rs

```diff
@@ -45,6 +42,15 @@ pub enum CoreStage {
     /// Name of app stage that runs after all other app stages
     Last,
 }
+
+/// The label for the Startup [`Schedule`](bevy_ecs::schedule::Schedule),
+/// which runs once at the beginning of the app.
+///
+/// Consists of the sub-stages defined in [`StartupStage`]. Systems added here are
+/// referred to as "startup systems".
+#[derive(Debug, Hash, PartialEq, Eq, Clone, StageLabel)]
+pub struct StartupSchedule;
```

Comment:
Maybe it could be not `pub` to make sure it's only access is through the official methods from `App`?

Reply:
I think theres no real harm in giving people direct access to the startup schedule if they want it. I think the "foot guns" have been resolved in this pr, and that gives "feature parity" with the stages.